docs/src/main/paradox/docs/fusion/architecture.md

```diff
@@ -2,30 +2,32 @@
 
 ## Technology Overview
 
-Nexus Fusion is a server-side rendered single-page webapp powered by @link:[React.js](https://react.dev){ open=new }.
-It is written in @link:[Typescript](https://www.typescriptlang.org/){ open=new }, and therefore must be transpiled
+Nexus Fusion is a partially server-side rendered single-page web app powered by @link:[React.js](https://react.dev){ open=new }.
+It is written in @link:[TypeScript](https://www.typescriptlang.org/){ open=new }, and therefore must be transpiled
 into native browser JavaScript, during a build step, before being served.
 
-The build step produces a server artifact to run on a Node.js server, with client-side assets and Javascript.
+The build step produces a server artifact to run on a Node.js server, with client-side assets and JavaScript.
 
-We produce a docker image that is able to serve the compiled assets directly, which is available on
+We produce a Docker image that is able to serve the compiled assets directly, which is available on
 @link:[Dockerhub](https://hub.docker.com/repository/docker/bluebrain/nexus-web){ open=new }.
 
 Although the application is served by a Node.js server, the client communicates directly to @ref:[Nexus Delta](../delta/index.md) using @ref:[Nexus.js](../utilities/index.md#nexus-js).
 
 ## Pages
 
-Nexus Fusion has undergone a significant restructuring to transition from a SubApp-based architecture to a page-based structure. This change will prepare the application for a full migration to file-system based routing technologie and help seperating the application main features.
+Nexus Fusion has undergone a significant restructuring to transition from a SubApp-based architecture to a page-based structure. This change will prepare the application for a full migration to file-system based routing system and help separating the application main features.
```

Comment:
For some reason "...and help **separate** the **application's** main features", reads better to me.

Reply:
#4533